datastore/cloud-client/src/main/java/com/example/datastore/QuickstartSample.java

```diff
@@ -46,4 +46,3 @@ public static void main(String... args) throws Exception {
     System.out.printf("Saved %s: %s%n", task.getKey().getName(), task.getString("description"));
   }
 }
-// [END datastore_quickstart]
```

Comment:
All fixed. You are right, some of them were mistakes. Some of the deletions are legit though.